mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.java

```diff
@@ -101,7 +101,7 @@ private static void scheduleWorker(Context context, String key) {
         enqueueItemUpload(key, prefix + setting.second, getter.getValue(context));
     }
 
-    private static void enqueueItemUpload(String tag, String itemName, String value) {
+    public static void enqueueItemUpload(String tag, String itemName, String value) {
```

Comment:
Thinking about it, I'd prefer to leave this private (and the prefix constant package private) and introducing a new method `public static void enqueueNfcUpdate(NfcTag tag)` which does the necessary wrapping.

(Reason is that IMHO the public methods should guide the callers in how they're to be used)